tests/materialize/materialize-bigquery/setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+# Ensure canonical Postgres environment variables are set.
+export PROJECT_ID="${GCP_BQ_PROJECT_ID}"
+export DATASET="${GCP_BQ_DATASET}"
+export REGION="${GCP_BQ_REGION}"
+export BUCKET="${GCP_BQ_BUCKET}"
+export BASE_64_CREDENTIALS_JSON=$(echo ${GCP_SERVICE_ACCOUNT_KEY} | base64)
+
+config_json_template='{
+   "project_id":        "$PROJECT_ID",
+   "dataset":           "$DATASET",
+   "region":            "$REGION",
+   "bucket":            "$BUCKET",
+   "credentials_json":  "$BASE_64_CREDENTIALS_JSON"
+}'
+
+resources_json_template='[
+  {
+    "resource": {
+      "table": "simple"
+    },
+    "source": "${TEST_COLLECTION_SIMPLE}"
+  },
+  {
```

Comment:
A case that is absent: Delta updates.

I found that testing a case like this (say, duplicate keys with delta updates) can be shown to work correctly 100% of the time, but it is cumbersome to verify the results. Timing issues make it so that sometimes the flow runtime sends a higher number of more granular delta updates than others where it sends a smaller number of "condensed" updates. This can vary apparently based on network conditions, or even which bindings are included in the test (their ordering). So I left that case out, but tested it quite a bit manually to be sure it works.